services/QuillLMS/engines/evidence/lib/evidence/evidence/profanity.rb

```diff
@@ -0,0 +1,26 @@
+module Evidence 
+  class Profanity 
+    def self.profane?(word)
+      return false unless word.respond_to?(:length) && word.length > 1
```

Comment:
is the `respond_to?(:length)` checking for objects outside of `String` (maybe symbols)?  If not, would word.is_a?(String) be more appropriate?  Otherwise, it seems like you would need similar methods for `word.respond_to?(:downcase)` and `word.respond_to?(:match)`